RocketChat/Rocket.Chat
# fix: importers progress bar

## Proposed changes (including videos or screenshots)


Inaccurate progress bar for selected import items: the import will finish at 2% of the progress bar.

* Change the progress bar component to use the fuselage version: `ProgressBar`
* Changed how the progress is updated on the `ImportProgressPage.tsx` file, in order to improve the UX and fix the issue.

![progress-bar](https://github.com/RocketChat/Rocket.Chat/assets/84182706/95b67db9-de2c-4f5f-9dab-c45de76693a4)



## Issue(s)

Task on Jira: [WM-122](https://rocketchat.atlassian.net/browse/WM-122?atlOrigin=eyJpIjoiMjMzOTc1NmZiOTljNDI2MThlZDUyMTgyNDVhMDgzMWIiLCJwIjoiaiJ9)
## Steps to test or reproduce

- Open Rocket.Chat
- Click on “Workspace” → Import
- Click on “Import new file”
- Select the import option (e.g: Slack)
- Upload a file
- In the table, remove the selection
- Select some channels
- Click to Import

*Expected Behavior:*
The progress bar should account only for the select data.

*Actual Behavior:*
The progress bar is counting all the files, so it’s possible that the import finishes at 2% which is not accurate since the 2% is for the full file and not for the selected ones.

## Further comments

WM-122

[WM-122]: https://rocketchat.atlassian.net/browse/WM-122?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
